controllers/hvpa_controller.go

```diff
@@ -693,7 +694,7 @@ func getWeightedReplicas(hpaStatus *autoscaling.HorizontalPodAutoscalerStatus, h
 		return outHpaStatus, err
 	}
 
-	lastScaleTime := hvpa.Status.LastScaling.LastScaleTime.DeepCopy()
+	lastScaleTime := hvpa.Status.LastScaling.LastUpdated.DeepCopy()
 	overrideScaleUpStabilization := hvpa.Status.OverrideScaleUpStabilization
 	if overrideScaleUpStabilization {
```

Comment:
Shouldn't we still ignore weight in case of `overrideScaleUpStabilization`?

Reply:
The thought had crossed my mind as well, but decided against it :)

Considering following cases:
1. hpaWeight == 0. So vpaWeight == 100. the memory requirement would be fulfilled by a full vertical scaling.
1. hpaWeight > 0. There should be no need to override weight in this case, since both VPA and HPA recommendations will be taken into account
1. If `spec.vpa.deploy` is set to `false`, or the vpa mode is set to `Off`, then it should be upto the user to set memory as a parameter for HPA scaling if he/she encounters any issue

Let me know if this makes sense, or I have missed something.

Reply:
I was just asking. Makes sense.